core/unix/signal.c

```diff
@@ -7760,7 +7760,7 @@ sig_detach_go_native(sig_detach_info_t *info)
     ASSERT_NOT_REACHED();
 }
 
-/* Sets this (slave) thread to detach by directly returning from the signal. */
+/* Sets this (worker) thread to detach by directly returning from the signal. */
```

Comment:
Hmm, to me "worker" implies a thread pool or something of threads dedicated to a task, while this is an app thread but is just not the one orchestrating the detach. Maybe "secondary" or "follower" instead?

Reply:
Done; changed to "secondary".